site/docs/api-quickstart.md

```diff
@@ -108,14 +110,32 @@ To create an Iceberg schema from an existing Avro schema, use converters in `Avr
 
 ```scala
 import org.apache.iceberg.avro.AvroSchemaUtil
+import org.apache.iceberg.Schema
 import org.apache.avro.Schema.Parser
+import org.apache.iceberg.types.Type
+
+val avroSchemaString = """{
+	"type": "record",
+	"name": "person",
+	"fields": [{
+		"name": "name",
+		"type": ["string", "null"]
+	}]
+}
+"""
+
+val avroSchema = new Parser().parse(avroSchemaString)
+
+val schemaType: Type = AvroSchemaUtil.convert(avroSchema)
+
+val fields = schemaType.asNestedType().asStructType().fields()
 
-val avroSchema = new Parser().parse(
-    """{ "type": "record", "name": "com.example.AvroType",
-      |  "fields": [ ... ]
-      |}""".stripMargin
+val icebergSchema = new Schema(fields)
```

Comment:
Thanks for fixing this. Unfortunately, it makes the example quite a bit longer. Maybe we should add `AvroSchemaUtil.convertSchema` that returns an Iceberg `Schema` instead of `Type`. That's what I intended here.

Reply:
I like the suggestion, done!